.gitignore

```diff
@@ -57,3 +57,22 @@ typings/
 # dotenv environment variables file
 .env
 
+# jvm bits
+*.class
+*.log
+
+# ignore files 
+docker/logs/*
+# but keep the folder
+!docker/logs/.keep
+
+
+# gradle etc.
+.gradle/*
+.idea/*
+clueso.iml
+out/*
+src/main/main.iml
+src/test/test.iml
+build/*
+
```

Comment:
add gradle/wrapper/gradle-wrapper.jar? for some reason appears below

Reply:
This is normal, they are unlikely to change and easier for the ones that don't have gradle installed – although it may seem odd, this is a good way to control which version of gradle should be run and the wrapper itself takes just a few KB (53.4 KB to be precise)